netbox/dcim/tables/racks.py

```diff
@@ -63,11 +63,15 @@ class RackTypeTable(NetBoxTable):
         verbose_name=_('Height')
     )
     outer_width = tables.TemplateColumn(
-        template_code="{{ record.outer_width }} {{ record.outer_unit }}",
+        template_code=OUTER_UNIT,
         verbose_name=_('Outer Width')
     )
+    outer_height = tables.TemplateColumn(
```

Comment:
We can use our own custom `columns.TemplateColumn` here, which automatically renders a placeholder for null values, simplifying the template code.